lib/opensusebasetest.pm

```diff
@@ -376,24 +376,24 @@ sub select_bootmenu_more {
 }
 
 sub export_kde_logs {
-    select_console 'log-console';
     save_screenshot;
-
-    if (check_var("DESKTOP", "kde")) {
-        if (get_var('PLASMA5')) {
-            my $fn  = '/tmp/plasma5_configs.tar.bz2';
-            my $cmd = sprintf 'tar cjf %s /home/%s/.config/*rc', $fn, $username;
-            type_string "$cmd\n";
-            upload_logs $fn;
-        }
-        else {
-            my $fn  = '/tmp/kde4_configs.tar.bz2';
-            my $cmd = sprintf 'tar cjf %s /home/%s/.kde4/share/config/*rc', $fn, $username;
-            type_string "$cmd\n";
-            upload_logs $fn;
-        }
-        save_screenshot;
+    return unless check_var('DESKTOP', 'kde');
```

Comment:
Shouldn't this be before `save_screenshot;`

Reply:
true however I simply have not changed the location. But with removal of the `select_console` call I should probably just do that.